pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -197,23 +197,23 @@ func (pl *CSILimits) getCSIDriverInfo(csiNode *storagev1.CSINode, pvc *v1.Persis
 
 		pluginName, err := pl.translator.GetInTreePluginNameFromSpec(pv, nil)
 		if err != nil {
-			klog.V(5).Infof("Unable to look up plugin name from PV spec: %v", err)
+			klog.V(5).Infof("Unable to look up plugin name from PV spec", "PV", klog.KObj(pv), "err", err)
```

Comment:
ditto, @cofyc is this expected to happen? should we use an ErrorS here?

Reply:
I'm not sure, cc @msau42

Reply:
Since we made it past the IsPVMigratable check above, then this check is expected to succeed.